src/main/java/duke/UI.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * The UI class handles user interactions and printing messages in the Duke application.
+ */
+public class UI {
+    private final String logo = "       ___\n"
+            + "    . -^   `--,\n"
+            + "   /# =========`-_\n"
+            + "  /# (--====___====\\\n"
+            + " /#   .- --.  . --.|\n"
+            + "/##   |  * ) (   * ),\n"
+            + "|##   \\    /\\ \\   / |\n"
+            + "|###   ---   \\ ---  |\n"
+            + "|####      ___)    #|\n"
+            + "|######           ##|\n"
+            + " \\##### ---------- /\n"
+            + "  \\####           (\n"
+            + "   `\\###          |\n"
+            + "     \\###         |\n"
+            + "      \\##        |\n"
+            + "       \\###.    .)\n"
+            + "        `======/";
```

Comment:
I like that you have your own logo! Very cool! :) 
ps: I am a Rick and Morty fan as well. Pickle Rick!